src/main/kotlin/com/hibob/academy/resource/PetResource.kt

```diff
@@ -45,4 +45,20 @@ class PetResource(
     fun deletePet(@PathParam("petId") petId: Long) : Response {
         return Response.ok().build()
```

Comment:
No need to implement yet?

Reply:
No need to implement it yet. but I can write it